azure-automation-plus-dsc-lab/nested/14.17.00.createDevServer.json

```diff
@@ -0,0 +1,139 @@
+﻿{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "devPrefix": {
+      "type": "string"
+    },
+    "azureRegion": {
```

Comment:
make sure we're using "location" as the name of the location parameter throughout